### PR TITLE
PLFM-6957

### PR DESCRIPTION
**Pull request by xschildw**

### CODE REVIEW

File: lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/AuthenticationMethod.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "description": "JSON enum for the types of authentication in Synapse",
+  "type": "string",
+  "enum": [
+    {
+      "name": "BASIC",
+      "description": "TODO: Auto-generated description"
```
Comment (brucehoff) [inline lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/AuthenticationMethod.json]: please fill in the descriptions

File: services/repository/src/test/java/org/sagebionetworks/repo/web/AccessInterceptorTest.java [hunk 2]
```diff
@@ -96,28 +107,26 @@ public void before() throws Exception {
...
 	}
 	
 	
 	
 	@Test
-	public void testHappyCase() throws Exception{
-		long start = testClock.currentTimeMillis();
+	public void testHappyCase() throws Exception {
+		when(mockRequest.getHeader(AuthorizationConstants.SYNAPSE_AUTHENTICATION_METHOD_HEADER_NAME)).thenReturn("SESSIONTOKEN");
+		//doReturn("SESSIONTOKEN").when(mockRequest.getHeader("authorizationMethod"));
```
Comment (brucehoff) [inline services/repository/src/test/java/org/sagebionetworks/repo/web/AccessInterceptorTest.java]: remove dead code

**Review by brucehoff — CHANGES_REQUESTED** [review]
Just a couple changes.  Overall it looks great.  I really like your refactor of AccessInterceptorTest.

**Comment by xschildw** [comment]
Wish I could take credit for that refactor but @marcomarasca guided me through it. I agree it's elegant...

**Pull request merged**